packages/compass-aggregations/src/utils/pipeline-storage.ts

```diff
@@ -67,14 +67,18 @@ function hasAllRequiredKeys(pipeline?: any): pipeline is StoredPipeline {
 
 export class PipelineStorage {
   async loadAll(): Promise<StoredPipeline[]> {
-    const dir = getDirectory();
-    const files = (await fs.readdir(dir))
-      .filter((file) => file.endsWith('.json'))
-      .map((file) => path.join(dir, file));
-
-    return (
-      await Promise.all(files.map((filePath) => this._loadOne(filePath)))
-    ).filter(Boolean) as StoredPipeline[];
+    try {
+      const dir = getDirectory();
```

Comment:
Will create a follow up to inject `basepath` here as well and remove `getDirectory` dependency